src/main/java/io/odpf/depot/bigquery/handler/JsonErrorHandler.java

```diff
@@ -62,31 +89,27 @@ private List<Entry<Long, List<BigQueryError>>> getUnknownFieldBqErrors(Map<Long,
                 .filter((x) -> {
                     List<BigQueryError> value = x.getValue();
                     List<BigQueryError> bqErrorsWithNoSuchFields = getBqErrorsWithNoSuchFields(value);
-                    if (!bqErrorsWithNoSuchFields.isEmpty()) {
-                        return true;
-                    }
-                    return false;
-
+                    return !bqErrorsWithNoSuchFields.isEmpty();
                 }).collect(Collectors.toList());
         return unkownFieldFieldBqError;
```

Comment:
can we inline this?
```
  private List<Entry<Long, List<BigQueryError>>> getUnknownFieldBqErrors(Map<Long, List<BigQueryError>> insertErrors) {
        return insertErrors.entrySet().parallelStream()
                .filter((x) -> {
                    List<BigQueryError> value = x.getValue();
                    List<BigQueryError> bqErrorsWithNoSuchFields = getBqErrorsWithNoSuchFields(value);
                    return !bqErrorsWithNoSuchFields.isEmpty();
                }).collect(Collectors.toList());
    }
    ```

Reply:
do you mean this ? 
`                     List<BigQueryError> value = x.getValue();
-                    List<BigQueryError> bqErrorsWithNoSuchFields = getBqErrorsWithNoSuchFields(value);
-                    return !bqErrorsWithNoSuchFields.isEmpty();
+                    return !getBqErrorsWithNoSuchFields(value).isEmpty();`

Reply:
Do you mean this ?
```
-                    List<BigQueryError> bqErrorsWithNoSuchFields = getBqErrorsWithNoSuchFields(value);
-                    return !bqErrorsWithNoSuchFields.isEmpty();
+                    return !getBqErrorsWithNoSuchFields(value).isEmpty();
```

Reply:
refactored, removed the variable.